srcpkgs/font-inter/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'font-inter'
+pkgname=font-inter
+version=3.19
+revision=1
+create_wrksrc=yes
+depends="font-util"
+short_desc="Variable typeface carefully crafted & designed for computer screens"
+maintainer="Robin Lundgren <linkert@onan.in>"
+license="OFL-1.1"
+homepage="https://rsms.me/inter/"
+distfiles="https://github.com/rsms/inter/releases/download/v${version}/Inter-${version}.zip"
+checksum=150ab6230d1762a57bebf35dfc04d606ff91598a31d785f7f100356ecdcc0032
+
+font_dirs="/usr/share/fonts/OTF"
+
+do_install() {
+	vmkdir usr/share/fonts/OTF
+	vcopy *Desktop/*.otf usr/share/fonts/OTF
+	vlicense LICENSE.txt LICENSE.md
+}
```

Comment:
_I_ (personally) think it's better to not put too much fonts into `/usr/share/fonts/OTF`, but split them into sub-directory by category, publisher, etc...

Reply:
Not at all against this suggestion, but I don't think I have found any other font package considering this? Therefore I decided to simply follow along. Do you know of any?

If we are going to organize by sub-categories there should probably be some syntax, some pattern right?

Overkill?:
```
../OTF/variable/sans/rsms/inter/*
../OTF/static/sans/{publisher}/foo/*
../OTF/static/monospace/{publisher}/bar/*
```

Reply:
I would just put them in /usr/share/fonts/inter